.travis.yml

```diff
@@ -14,16 +14,20 @@ env:
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Debug
     - TESTING=OFF CMAKE_BUILD_TYPE=Debug   WERROR=yes
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release DOCKERHUB=yes
+    - DISTRO=fedora_gmx5.1 TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release
```

Comment:
We could label the test here a xtp-test:
```cmake
SET_TESTS_PROPERTIES( ${PROG}Help PROPERTIES LABEL "xtp" )
```
and well as `votca` and then use `TESTOPTS="-L xtp"` to only run those...